src/_posts/databases/postgresql/2000-01-01-managing.md

```diff
@@ -102,7 +102,7 @@ your data and fits your application workload.
 3. If so, run `terraform apply`
 
 
-## Upgrading
+## Upgrade
```

Comment:
I really wonder about this change (and for all the title changed in this PR). The titles are using present continous tense (at least in PG category since François's refacto).
That's why, for the sake of consistency, I'd keep the title as they are.

Reply:
Okay I'll remove them for consistency, but the use of "ing" everywhere is a bad english practice